packages/xstate-react/src/fsm.ts

```diff
@@ -88,11 +88,9 @@ export function useService<
   StateMachine.Service<TContext, TEvent, TState>['send'],
   StateMachine.Service<TContext, TEvent, TState>
 ] {
-  const subscription: Subscription<StateMachine.State<
-    TContext,
-    TEvent,
-    TState
-  >> = useMemo(() => {
+  const subscription: Subscription<
+    StateMachine.State<TContext, TEvent, TState>
+  > = useMemo(() => {
```

Comment:
I thought about excluding this change but it was generated by the pre-commit hook, so I wasn't sure what I should do. Happy to push a commit reverting this line and not run the pre-commit hook.